docs/UserGuide.md

```diff
@@ -1,35 +1,25 @@
-# AddressBook Level 4 - User Guide
+# TuLuist - User Guide
 
-By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbsp;&nbsp; Licence: `MIT`
+By : `Team TuLuist`  &nbsp;&nbsp;&nbsp;&nbsp;
 
 ---
 
 1. [Quick Start](#quick-start)
 2. [Features](#features)
-3. [FAQ](#faq)
-4. [Command Summary](#command-summary)
+3. [Command Summary](#command-summary)
 
 ## 1. Quick Start
 
-0. Ensure you have Java version `1.8.0_60` or later installed in your Computer.<br>
+1. Double-click the file to start the app. The GUI should appear in a few seconds.
 
-   > Having any Java 8 version is not enough. <br>
-   > This app will not work with earlier versions of Java 8.
-
-1. Download the latest `addressbook.jar` from the [releases](../../../releases) tab.
-2. Copy the file to the folder you want to use as the home folder for your Address Book.
-3. Double-click the file to start the app. The GUI should appear in a few seconds.
-   > <img src="images/Ui.png" width="600">
-
-4. Type the command in the command box and press <kbd>Enter</kbd> to execute it. <br>
+2. Type the command in the command box and press <kbd>Enter</kbd> to execute it. <br>
    e.g. typing **`help`** and pressing <kbd>Enter</kbd> will open the help window.
-5. Some example commands you can try:
-   * **`list`** : lists all contacts
-   * **`add`**` John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01` :
-     adds a contact named `John Doe` to the Address Book.
-   * **`delete`**` 3` : deletes the 3rd contact shown in the current list
+3. Some example commands you can try:
+   * **`add`**` Try Out Todo List  d/11-12-2011` :
+     adds a task named `Try Out Todo List` to the todo list.
```

Comment:
Not sure what you mean here by mismatched example.

Also, parameters can be entered in any order, so it doesn't matter which one is listed first.